src/lib.rs

```diff
@@ -3,12 +3,16 @@
 //!
 //! See examples directory for examples.
 
-#![deny(missing_docs)]
+#![warn(missing_docs)]
```

Comment:
If you are going for 1.0 you might want to keep deny.

Reply:
I disagree, 1.0 doesn't mean no more development is going to happen it just indicates the code is considered "production ready".

Reply:
which is why I think docs are essential for "production ready"

Reply:
Changing to warn during development is fine, but I usually turn it back to deny when pushing code.

Reply:
Warnings should all be resolved by the time a PR is merged, if they aren't then the developers weren't careful enough imo.  Keeping a deny(missing_docs) makes prototyping more difficult and doesn't even work.  Most of the time this causes people to override the compiler by writing

```rust
/// 
pub fn thing()
```

This happened to the amethyst renderer at one point, and leaving that kind of "documentation" in without a warning is a much bigger problem.

Reply:
warn is fine, deny is too restrictive imho

Reply:
okay